### DISCUSSION TITLE
Create New Durables  i want to set the value with if condition

### DISCUSSION THREAD

**Discussion by gettosam** [postmanlabs/postman-flows#296] [Q&A] [answered]
Hello,
I have a scenario, where I want to set the value for a variable in the Durable block, with the value from the previous data request. and I want to check if the value is empty, default the variable with a default value .

Is this feature exist ? if yes, how do i do this with "Create Durable" block ?

**Comment by samay-v** [comment, marked as answer]
Hey @gettosam 
You can achieve this by attaching a `Condition` block with the `!is null` expression before setting your durables. 
![image](https://user-images.githubusercontent.com/28357106/163020047-42ed2131-53ce-4346-8c9a-df905b5c7437.png)

Make sure the keys of the durables are the same in both the blocks, this way the request will be able to use the input from whichever durable was triggered.

**Reply by gettosam** [reply]
Thank you, i would love the Postman flow team add javascript or ternary operator like

( /data/body/args/foo !is null) ? ( /data/body/args/foo ) : "Defaultvalue"
